_search-plugins/neural-search.md

```diff
@@ -10,8 +10,8 @@ redirect_from:
 
 # Neural Search plugin
 
-The Neural Search plugin is an experimental feature. For updates on the progress of the Neural Search plugin, or if you want to leave feedback that could help improve the feature, join the discussion in the [Neural Search forum](https://forum.opensearch.org/t/feedback-neural-search-plugin-experimental-release/11501).    
-{: .warning}
+The Neural Search plugin was taken out of experimental status and released as Generally Available in OpenSearch 2.9.  
```

Comment:
Do we need to say that it was taken out of experimental status? I'm not certain that the user needs that information. Would just "The Neural Search plugin has been released as Generally Available in OpenSearch 2.9" suffice?

Reply:
Or just "The Neural Search plugin is Generally Available as of OpenSearch 2.9"?

Reply:
Changing now.